lib/portage/repository/config.py

```diff
@@ -177,6 +179,13 @@ def __init__(self, name, repo_opts, local_config=True):
 		else:
 			masters = None
 
+		self.maintainer_nag = repo_opts.get('maintainer-nag', 'yes').lower() == 'yes'
+
+		maintainer_nag_url = repo_opts.get('maintainer-nag-url')
```

Comment:
We should also support setting maintainer-nag-url in metadata/layout.conf, which means supporting it in parse_layout_conf and getting the value from layout_data.